stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ChatManager.kt

```diff
@@ -0,0 +1,73 @@
+package io.getstream.chat.android.compose.sample
+
+import android.content.Context
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.errors.ChatError
+import io.getstream.chat.android.client.logger.ChatLogLevel
+import io.getstream.chat.android.client.notifications.handler.NotificationConfig
+import io.getstream.chat.android.client.notifications.handler.NotificationHandlerFactory
+import io.getstream.chat.android.compose.sample.data.UserCredentials
+import io.getstream.chat.android.compose.sample.ui.StartupActivity
+import io.getstream.chat.android.core.ExperimentalStreamChatApi
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+import io.getstream.chat.android.offline.experimental.plugin.Config
+import io.getstream.chat.android.offline.experimental.plugin.OfflinePlugin
+import io.getstream.chat.android.pushprovider.firebase.FirebasePushDeviceGenerator
+
+/**
+ * A helper class that is responsible for initializing the SDK and connecting/disconnecting
+ * a user. Under the hood, it persists the user so that we are able to connect automatically
+ * next time the app is launched.
+ */
+@OptIn(InternalStreamChatApi::class)
+class ChatManager(private val context: Context) {
+
+    @OptIn(ExperimentalStreamChatApi::class)
```

Comment:
I know it's not specifically SDK-based code but some docs on these functions could be good (just for our sanity)